examples/with-custom-compress-encodings/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import Link from 'next/link'
+
+export default () => (
+  <div>
+    <Link href='/'>
+      <a style={styles.a} >Home</a>
+    </Link>
+
+    <Link href='/about'>
+      <a style={styles.a} >About</a>
```

Comment:
I just wanna talk about this feature in this example.
Anyway, I don't mind using that too.